### PR TITLE
CPLP-1397: Org specific package names

### PR DESCRIPTION
**Pull request by evegufy**
https://jira.catena-x.net/browse/CPLP-1397
Necessary as preparation for (FOSS) migration to Eclipse Foundation.

### CODE REVIEW

**Comment by evegufy** [comment]
@ntruchsess @Phil91 Could you please give the input for the 2 Vulnerabilities and the 1 Security Hotspot in SonarCloud? I assume you did document (not solve) those as part of https://jira.catena-x.net/browse/CPLP-1454 already, I think I need to redo that documentation.

**Comment by Phil91** [comment]
> @ntruchsess @Phil91 Could you please give the input for the 2 Vulnerabilities and the 1 Security Hotspot in SonarCloud? I assume you did document (not solve) those as part of https://jira.catena-x.net/browse/CPLP-1454 already, I think I need to redo that documentation.

@evegufy yes the Security Hotspot isn't a problem since we have it configurable and only enable it when needed. So on production one wouldn't set the configuration to true, therefor no one can see the developer exception pages.

Regarding the two Vulnerabilities, the configuration will only be set for our local development environment to bypass our certificate problem.

I'm pretty sure i've already commented and set the 3 issues to solved with the Tasks [CPLP-1333](https://jira.catena-x.net/browse/CPLP-1333) & [CPLP-1334](https://jira.catena-x.net/browse/CPLP-1334)

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: docker/Dockerfile-administration-service [hunk 1]
```diff
@@ -16,4 +16,4 @@ ENV ASPNETCORE_URLS http://+:8080
 EXPOSE 8080
 RUN chown -R 1000:3000 /app
 USER 1000:3000
-ENTRYPOINT ["dotnet", "CatenaX.NetworkServices.Administration.Service.dll"]
+ENTRYPOINT ["dotnet", "Administration.Service.dll"]
```
[the remark this answers is not in the document]
Comment (evegufy) [reply docker/Dockerfile-administration-service]: done and successfully tested

File: docker/Dockerfile-maintenance-service [hunk 2]
```diff
@@ -18,4 +18,4 @@ ENV ASPNETCORE_URLS http://+:8080
 EXPOSE 8080
 RUN chown -R 1000:3000 /app
 USER 1000:3000
-ENTRYPOINT ["dotnet", "CatenaX.NetworkServices.Maintenance.App.dll"]
+ENTRYPOINT ["dotnet", "Maintenance.App.dll"]
```
[the remark this answers is not in the document]
Comment (evegufy) [reply docker/Dockerfile-maintenance-service]: done

File: docker/Dockerfile-services-service [hunk 3]
```diff
@@ -16,4 +16,4 @@ ENV ASPNETCORE_URLS http://+:8080
 EXPOSE 8080
 RUN chown -R 1000:3000 /app
 USER 1000:3000
-ENTRYPOINT ["dotnet", "CatenaX.NetworkServices.Services.Service.dll"]
+ENTRYPOINT ["dotnet", "Services.Service.dll"]
```
[the remark this answers is not in the document]
Comment (evegufy) [reply docker/Dockerfile-services-service]: done

File: docker/Dockerfile-notification-service [hunk 4]
```diff
@@ -16,4 +16,4 @@ ENV ASPNETCORE_URLS http://+:8080
 EXPOSE 8080
 RUN chown -R 1000:3000 /app
 USER 1000:3000
-ENTRYPOINT ["dotnet", "CatenaX.NetworkServices.Notification.Service.dll"]
+ENTRYPOINT ["dotnet", "Notification.Service.dll"]
```
[the remark this answers is not in the document]
Comment (evegufy) [reply docker/Dockerfile-notification-service]: done

File: docker/Dockerfile-provisioning-service [hunk 5]
```diff
@@ -16,4 +16,4 @@ ENV ASPNETCORE_URLS http://+:8080
 EXPOSE 8080
 RUN chown -R 1000:3000 /app
 USER 1000:3000
-ENTRYPOINT ["dotnet", "CatenaX.NetworkServices.Provisioning.Service.dll"]
+ENTRYPOINT ["dotnet", "Provisioning.Service.dll"]
```
[the remark this answers is not in the document]
Comment (evegufy) [reply docker/Dockerfile-provisioning-service]: done

File: docker/Dockerfile-registration-service [hunk 6]
```diff
@@ -16,4 +16,4 @@ ENV ASPNETCORE_URLS http://+:8080
 EXPOSE 8080
 RUN chown -R 1000:3000 /app
 USER 1000:3000
-ENTRYPOINT ["dotnet", "CatenaX.NetworkServices.Registration.Service.dll"]
+ENTRYPOINT ["dotnet", "Registration.Service.dll"]
```
[the remark this answers is not in the document]
Comment (evegufy) [reply docker/Dockerfile-registration-service]: done

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**